<launch>
    <rosparam file="$(find suitbot_ros)/config/suitbot.yaml" />

    <param name="map_file" value="$(find suitbot_ros)/data/wean_map_grid_edited.jpg"/>
    <param name="model_path" value="$(find suitbot_ros)/scripts/audio/vosk-model-small-en-us-0.15"/>

    <node name="job_management" pkg="suitbot_ros" type="job_management" output="screen"/>
    <node name="planner_node" pkg="suitbot_ros" type="planner_node" output="screen"/>
    
    <node name="tracking_simulator" pkg="suitbot_ros" type="tracking_simulator.py" output="screen"/>
    <node name="embedded_device_driver" pkg="suitbot_ros" type="embedded_device_driver.py" output="screen"/>
    <node name="audio_listener" pkg="suitbot_ros" type="audio_listener.py" output="screen"/>
    <node name="audio_speaker" pkg="suitbot_ros" type="audio_speaker.py" output="screen"/>
    
</launch>